<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialEditor</class>
 <widget class="QWidget" name="MaterialEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>189</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QPushButton" name="m_closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="m_BlinnPhongGroup">
     <property name="title">
      <string>Material Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,0,0">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="kdR">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kdG">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kdB">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Diffuse Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="ksR">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ksG">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ksB">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Specular color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Shininess</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="exp">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1024.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>32.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="kUsePerVertex">
        <property name="text">
         <string>Per-vertex</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Ra::Gui::ColorWidget" name="kdColorWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Ra::Gui::ColorWidget" name="ksColorWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="m_renderObjectName">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ra::Gui::ColorWidget</class>
   <extends>QWidget</extends>
   <header location="global">Gui/ColorWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kUsePerVertex</sender>
   <signal>toggled(bool)</signal>
   <receiver>kdB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kUsePerVertex</sender>
   <signal>toggled(bool)</signal>
   <receiver>kdG</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kUsePerVertex</sender>
   <signal>toggled(bool)</signal>
   <receiver>kdR</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
